pkgs/tools/misc/ytmdl/default.nix

```
@@ -15,6 +16,15 @@ python3.pkgs.buildPythonApplication rec {
     sha256 = "1jpd5zhqg2m9vjjjw4mgzb594q1v1pq1yl65py6kw42bq9w5yl5p";
   };
 
+  patches = [
+    # Fixes ytmdl#188 and is only needed until the next major release after 2021.06.26
```

Please include a link to the upstream issue
```suggestion
    # Fixes https://github.com/deepjyoti30/ytmdl/issues/188
    # Only needed until the next major release after 2021.06.26
```